src/Cli/dotnet/commands/dotnet-test/LocalizableStrings.resx

```diff
@@ -261,4 +261,18 @@ This option is currently supported only on Windows together with netcoreapp2.1 a
   <data name="HangTimeoutArgumentName" xml:space="preserve">
     <value>TIMESPAN</value>
   </data>
+  <data name="CmdEnvironmentVariableDescription" xml:space="preserve">
+    <value>Sets the value of an environment variable. 
+Creates the variable if it does not exist, overrides if it does. 
+This will force the tests to be run in an isolated process. 
+This argument can be specified multiple times to provide multiple variables.
+
+Examples:
+-e VARIABLE1=VALUE1
+-e ANOTHER_VARIABLE="VALUE WITH SPACES"
+-e ANOTHER_VARIABLE="VALUE;seperated with;semicolons"</value>
```

Comment:
Consider naming the variables the same for the examples showing just one being provided in different formats, and adding an example for the multiple variable specification. I would also convert the values to lowercase. And maybe use "value;with;semicolon", to not mix semicolon and spaces.

```
-e VARIABLE=abc
-e VARIABLe="value with spaces"
-e VARIABLE="value;with;semicolons"
-e VAR1=abc -e VAR2=def -e VAR3=ghi
```

Reply:
I actually wanted show that you can mix spaces with semicolons.

Reply:
Okay. Let's use that then.